### DISCUSSION TITLE
📣 Material Web Components are now beta!

### DISCUSSION THREAD

**Discussion by asyncliz** [material-components/material-web#4527] [Announcements]
**tl;dr MWC is now beta and 1.0 is releasing 2023 Q3!**

Hello everyone,

We're excited to announce the beta release of Material Web Components!

**What does "beta" mean?**

- Stable APIs
- Tested
- ARIA accessible

Most notably, there should be fewer breaking changes. That means it's a good time for projects to start experimenting!
This is an important step towards our 1.0 stable release... and we're getting closer to a release date!! 🥳

✨ _MWC 1.0 will release this quarter in 2023 Q3_ ✨

😱😱😱

All 22 components from our [roadmap](https://github.com/material-components/material-web/#roadmap) are ready to start stress testing, and we would love feedback from you all!

**What's new and improved from the old [<mwc-*>](https://github.com/material-components/material-web/tree/mwc) components?**

- 🎨 Dynamic color and Material tokens
- 📋 Native HTML form support
- ♿️ Improved accessibility
- 🛠️ Bug fixes and more!

We encourage old MWC users to try out the new Material 3 components.

We're excited to see what you build! 🥰

The Material Web team

**Comment by borisdelev** [comment]
U make my day! Well done!
...may the force be with you. Always :)

**Reply by Arun-ez** [reply]
Eagerly waiting for it 🤩

**Reply by hughmancoder** [reply]
So excited for this release. I've seen new ui updates on google apps and it looks absolutely aesthetic af

**Comment by FluorescentHallucinogen** [comment]
Is there a page where I can play with all the components live?

**Reply by e111077** [reply]
We are currently working on filling out the catalog as well as many other features of it. It is currently on our GitHub pages and subject to move infrastructure and domain name:

https://material-components.github.io/material-web/

If anyone is interested in helping us fill it out, see this discussion on how to contribute:

https://github.com/material-components/material-web/discussions/4396#discussioncomment-6094512

We are looking to fill out docs pages for each of the components. If you'd like to contribute, shoot us a PR with the following requirements:

- follow the structure of the other docs in the `docs/` directory
- commit it to the `internal/` directory
- send us a PR which we will review and merge
- we will do some final polish to ensure it follows our style guide 100%
- we move it to `docs/` and everyone can enjoy it!

We also are looking to eventually add:

- a home page with info about the project
- a page with all the components rendered (perhaps into a playground?)
- an a11y pass (the components are accessible, but the site is not and that is my fault 🙈) 
- PR live catalog previews

If any of these features sounds interesting to you, please reach out either in Discussions of the #material chat in the [Lit Discord](https://lit.dev/discord) and we can include you into our Material Web developers Google Chat and also help you if you have any eng questions

**Comment by ismaeel066** [comment]
new
